mitodl/mit-open
# Add public learning paths to the index

# What are the relevant tickets?
Closes #106 

# Description (What does it do?)
- Creates, updates, or removes published learning path documents from the search index when appropriate

# How can this be tested?
- Create some learning paths if you don't have any already, with at least 1 public and 1 private, at this url (need to be logged in as an admin): http://localhost:8063/learningpaths/
- If you already have some, just run `./manage.py recreate_index --learning_paths`
- Check http://localhost:8063/learning_resources_search/?resource_type=learning_path, you should see published learning paths there only
- Edit a public learning path and make it private instead.
- Wait a minute, then go back to the search url above.  The learning path you made private should no longer show up.
- Edit a private path, making it public.
- Wait a minute, then go back to the search url above.  The learning path you made public should show up.
